doc/modules/classes.rst

```diff
@@ -760,47 +760,92 @@ Plotting
 
 .. currentmodule:: sklearn
 
+Linear classifiers
+------------------
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   linear_model.LogisticRegression
+   linear_model.LogisticRegressionCV
+   linear_model.PassiveAggressiveClassifier
+   linear_model.Perceptron
+   linear_model.RidgeClassifier
+   linear_model.RidgeClassifierCV
+   linear_model.SGDClassifier
+
+Traditional linear regressors
+-----------------------------
+
 .. autosummary::
    :toctree: generated/
    :template: class.rst
 
-   linear_model.ARDRegression
-   linear_model.BayesianRidge
    linear_model.ElasticNet
```

Comment:
It's strange to have separate categories for Lasso and ElasticNet

Reply:
they are in the same `Traditional linear regressors` category

Reply:
I think ElasticNet / Lasso models should be moved to the "Regressors with variable selection" section. But then LogisticRegression and SGDRegressor could also be moved there (depending on the penalty)....